src/share/poudriere/common.sh

```diff
@@ -1861,8 +1863,10 @@ load_blacklist() {
 	local setname=$3
 	local bl b bfile
 
-	bl="- ${setname} ${ptname} ${name} ${name}-${ptname}"
-	[ -n "${setname}" ] && bl="${bl} ${bl}-${setname} \
+	bl="- ${setname} ${ptname} ${name}"
+	[ -n "${setname}" ] && bl="${bl} ${ptname}-${setname}"
+	bl="${bl} ${name}-${ptname}"
+	[ -n "${setname}" ] && bl="${bl} ${name}-${setname} \
```

Comment:
Add I fixed this in `release-3.1` in 63b6b0618 but forgot to bring it into `master`.